presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -49,6 +51,7 @@ public class PrestoSparkConfig
     private int splitAssignmentBatchSize = 1_000_000;
     private double memoryRevokingThreshold;
     private double memoryRevokingTarget;
+    private List<String> executionStrategies = ImmutableList.of();
```

Comment:
Should we call it `queryExecutionStrategies` instead of just `executionStrategies`?